### DISCUSSION TITLE
Brand new user to brew and getting fatal error upon install

### DISCUSSION THREAD

**Discussion by Blurr3dP1x3L** [Homebrew/discussions#2890] [Getting started]
Hi Everyone!

I'm new to brew and coding, so I'm just trying to set up my mac with everything it needs including brew! I thought I had installed brew correctly, so when I asked it to update bash...well it didn't like that, and here are the following error messages!
 ```
" error: 5905 bytes of the body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: index-pack failed
Error: Failure while executing; `git clone https://github.com/Homebrew/homebrew-cask /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask --origin=origin --template=` exited with 128."
```

My MacBook pro is mid-2015
BigSur 11.4
2.2 GHz Quad-Core Intel Core i7
16 GB 1600 MHz DDR3
Intel Iris Pro 1536 MB

please help!

**Comment by SMillerDev** [comment]
See #666

**Reply by Blurr3dP1x3L** [reply]
Thanks, I actually already had a look at that, from my google search. I figured it out, the internet connection issue is actually because I had my vpn turned on, and GitHub rejected the connection. I'm surprised to see no one has thought of that but actually used a vpn to fix it....either way turn the vpn on or off!
